x-pack/test/functional/apps/ml/embeddables/anomaly_charts_dashboard_embeddables.ts

```diff
@@ -109,8 +110,52 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
           await PageObjects.timePicker.pauseAutoRefresh();
           await ml.dashboardEmbeddables.assertAnomalyChartsSeverityThresholdControlExists();
           await ml.dashboardEmbeddables.assertAnomalyChartsExists();
+          await PageObjects.dashboard.saveDashboard(testData.dashboardTitle);
         });
       });
     }
+
+    describe('supports migrations', function () {
+      const panelTitle = `Saved ML anomaly charts for fq_multi_1_ae`;
+      const dashboardSavedObject = {
+        attributes: {
+          title: `ML anomaly charts 7.15 dashboard ${Date.now()}`,
+          description: '',
+          panelsJSON: `[{"version":"7.15.2","type":"ml_anomaly_charts","gridData":{"x":0,"y":0,"w":36,"h":20,"i":"ffcdb1ed-0079-41ee-8dda-3f6c138182ab"},"panelIndex":"ffcdb1ed-0079-41ee-8dda-3f6c138182ab","embeddableConfig":{"jobIds":["fq_multi_1_ae"],"maxSeriesToPlot":6,"severityThreshold":0,"enhancements":{}},"title":"${panelTitle}"}]`,
+          optionsJSON: '{"useMargins":true,"syncColors":false,"hidePanelTitles":false}',
+          timeRestore: true,
+          timeTo: '2016-02-11T00:00:00.000Z',
+          timeFrom: '2016-02-07T00:00:00.000Z',
+          refreshInterval: {
+            pause: true,
+            value: 0,
+          },
+          kibanaSavedObjectMeta: {
+            searchSourceJSON: '{"query":{"query":"","language":"kuery"},"filter":[]}',
+          },
+        },
+        coreMigrationVersion: '7.15.2',
+      };
+
+      before(async () => {
+        await ml.testResources.createDashboardSavedObject(
+          dashboardSavedObject.attributes.title,
+          dashboardSavedObject
+        );
+        await PageObjects.common.navigateToApp('dashboard');
+      });
+
+      it('loads saved dashboard from version 7.15', async () => {
```

Comment:
Could the `coreMigrationVersion` from the saved object config be used in the test title?

Reply:
Updated here https://github.com/elastic/kibana/pull/116520/commits/eee3561c7c72a1cd85b7c65f4abc412935bd366f